ibm/service/vpc/resource_ibm_is_vpc_default_routing_table_test.go

```diff
@@ -0,0 +1,176 @@
+				),
+			},
+			{
+				Config: testAccCheckIBMISVPCDefaultRouteTableAcceptRoutesFromConfig(routeTableName1, name1, acceptRoutesFromVPNGateway),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckIBMISVPCDefaultRouteTableExists("ibm_is_vpc_default_routing_table.test_ibm_is_vpc_default_routing_table", vpcRouteTables),
+					resource.TestCheckResourceAttr(
+						"ibm_is_vpc_default_routing_table.test_ibm_is_vpc_default_routing_table", "name", routeTableName1),
+					resource.TestCheckResourceAttrSet(
+						"ibm_is_vpc_default_routing_table.test_ibm_is_vpc_default_routing_table", "accept_routes_from_resource_type.#"),
+				),
+			},
+			{
+				ResourceName:      "ibm_is_vpc_default_routing_table.test_ibm_is_vpc_default_routing_table",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccCheckIBMISVPCDefaultRouteTableDestroy(s *terraform.State) error {
+	//userDetails, _ := acc.TestAccProvider.Meta().(conns.ClientSession).BluemixUserDetails()
+
+	//sess, _ := acc.TestAccProvider.Meta().(conns.ClientSession).VpcV1API()
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "ibm_is_vpc_default_routing_table" {
+			continue
+		}
+		parts, err := flex.IdParts(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+		sess, err := vpcClient(acc.TestAccProvider.Meta())
+		if err != nil {
+			return err
+		}
+		vpcID := parts[0]
+		routeTableID := parts[1]
+		getVpcRoutingTableOptions := sess.NewGetVPCRoutingTableOptions(vpcID, routeTableID)
+		_, _, err = sess.GetVPCRoutingTable(getVpcRoutingTableOptions)
+		if err == nil {
+			return fmt.Errorf("Routing table still exists: %s", rs.Primary.ID)
+		}
+	}
+
+	return nil
+}
+
+func testAccCheckIBMISVPCDefaultRouteTableExists(n, vpcrouteTableID string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+
+		//sess, _ := acc.TestAccProvider.Meta().(conns.ClientSession).VpcV1API()
+
+		rs, ok := s.RootModule().Resources[n]
+
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return errors.New("No Record ID is set")
+		}
+
+		parts, err := flex.IdParts(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+
+		vpcID := parts[0]
+		routeTableID := parts[1]
+		sess, err := vpcClient(acc.TestAccProvider.Meta())
+		getVpcRoutingTableOptions := sess.NewGetVPCRoutingTableOptions(vpcID, routeTableID)
+		rtResponse, detail, err := sess.GetVPCRoutingTable(getVpcRoutingTableOptions)
+		if err != nil {
+			return fmt.Errorf("[ERROR] Error Getting Flow log: %s\n%s", err, detail)
+		}
+		vpcrouteTableID = *rtResponse.ID
+		return nil
+	}
+}
+
+func testAccCheckIBMISVPCDefaultRouteTableConfig(rtName, name string) string {
+	return fmt.Sprintf(`
+resource "ibm_is_vpc" "testacc_vpc" {
+	name = "%s"
+}
+resource "ibm_is_vpc_default_routing_table" "test_ibm_is_vpc_default_routing_table" {
+	depends_on = [ibm_is_vpc.testacc_vpc]
+	vpc = ibm_is_vpc.testacc_vpc.id
```

Comment:
Name and VPC as mentioned in the above comment, please change this accordingly.
and without VPC id getting the below error, please modify the acceptance testing accordingly..
<img width="888" alt="Screenshot 2023-02-17 at 10 36 52 AM" src="https://user-images.githubusercontent.com/78336632/219554183-6bb54434-1fae-4513-85ce-2c467123c717.png">